content/en/docs/instrumentation/cpp/getting-started.md

```diff
@@ -1,185 +1,393 @@
 ---
 title: Getting Started
+description: Get telemetry for your app in less than 5 minutes!
+cSpell:ignore: oatpp rolldice
 weight: 10
-# prettier-ignore
-cSpell:ignore: Bazel DBUILD devel DWITH helloworld libcurl openssl tracestate xcode
 ---
 
-Welcome to the OpenTelemetry C++ getting started guide! This guide will walk you
-through the basic steps in installing, instrumenting with, configuring, and
-exporting data from OpenTelemetry.
+This page will show you how to get started with OpenTelemetry in C++.
 
-You can use [CMake](https://cmake.org/) or [Bazel](https://bazel.build/) for
-building OpenTelemetry C++. The following getting started guide will make use of
-CMake and only provide you the most essential steps to have a working example
-application (a HTTP server & HTTP client). For more details read
-[these instructions](https://github.com/open-telemetry/opentelemetry-cpp/blob/main/INSTALL.md).
+You will learn how to instrument a simple C++ application automatically,
+such that [traces](/docs/concepts/signals/traces/), [metrics](/docs/concepts/signals/metrics/) and [logs](/docs/concepts/signals/logs/) are emitted to the terminal.
 
-## Prerequisites
+## Prerequisites 
 
-You can build OpenTelemetry C++ on Windows, macOS or Linux. First you need to
-install some dependencies:
+Ensure that you have the following installed locally:
 
-{{< tabpane text=true >}} {{% tab "Linux (apt)" %}}
+- Git
+- C++ compiler supporting C++ version >= 14
+- Make 
+- CMake version >= 3.1
```

Comment:
The latest master for `oatapp` seems to require CMake version [3.20](https://github.com/oatpp/oatpp/blob/17ef2a7f6c8a932498799b2a5ae5aab2869975c7/cmake/project.cmake#L1) or higher. The last released oatapp version `v.1.3.0` seems to build correctly with lower CMake version. Either update the CMake version here, or use the correct version of `oatapp`.

Reply:
```suggestion
- CMake version >= 3.20
```

I see now harm in raising that to 3.20, or is it unusual that people have that version?

Reply:
Yes, should be fine to upgrade CMake.

Reply:
@Akhaled19 apply that change suggested above to fix this.